lib/_http_server.js

```diff
@@ -305,20 +306,16 @@ ServerResponse.prototype.writeEarlyHints = function writeEarlyHints(hints, cb) {
 
   validateObject(hints, 'hints');
 
-  if (hints.link === null || hints.link === undefined) {
-    return;
-  }
-
-  const link = validateLinkHeaderValue(hints.link);
-
-  if (link.length === 0) {
-    return;
-  }
-
-  head += 'Link: ' + link + '\r\n';
-
   for (const key of ObjectKeys(hints)) {
-    if (key !== 'link') {
+    const hint = hints[key];
+    validateHeaderName(key);
+    validateHeaderValue(key, hint);
```

Comment:
Can we merge these two functions?

Reply:
They are used separately in `lib/_http_outgoing.js` so I don't think we can completely merge them. We could create a wrapper that calls both, but I'm not sure that's worth it, unless you disagree?

Reply:
I think this is ok